dashboard-server/queries/responses/token.js

```diff
@@ -0,0 +1,18 @@
+const base64 = require('base-64');
+const decode = base64.decode;
+const encode = base64.encode;
+
+class Token {
```

Comment:
The way you've implemented the class, it doesn't have any internal state. 

Why not just make this a module with "encode" and "decode" ?

Then instead of: 

```
const t = require('./token');
const token = new t.Token();
```

You can just do: 

```
const token = require('./token');
```